scripts/samples/add-file-to-collection.sh

```diff
@@ -1,11 +1,31 @@
 #!/bin/bash
 
+#!/bin/bash
+
+###################################################################
+#Script Name	: add-file-to-collection.sh
+#Description	: This is a script file that runs a curl command to add a file to the estuary content server.
```

Comment:
I think it's better to use Pestuary or any other SDK for this so we could to add the add, remove, list from the same python script.

Reply:
The point of the tests here is to ensure that the estuary endpoints will respond according to the raw request body we provide. I want to run test as raw (low-level) as possible by using `curl` and passing the correct request information (headers, auth, content-type). 

IMO, the scenarios you pointed here should all be unit tests in the `pestuary` repo.